sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/ft_personalize_3.c

```diff
@@ -29,6 +32,31 @@ enum {
       kAttestMeasurementSizeInBytes / sizeof(uint32_t),
 };
 
+static const flash_ctrl_perms_t kCertificateFlashInfoPerms = {
```

Comment:
nit: Should we disable write acl after we are done, or do we trust ROM_EXT to do so at boot time? The later is probably ok, but asking since we need to track this change for ROM_EXT.

Reply:
yes we should lock write in rom_ext. because it will get reset at boot anyway